blog.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adopt A Dog - Blog</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,700;0,800;0,900;1,900&family=Vollkorn:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="./css/normalize.css">
+    <link rel="stylesheet" href="./css/styles.css">
+  </head>
+  <body>
+    <header>
+      <div class="header-top row align-items-start justify-content-between">
+        <div class="col logo">
+          <img src="./images/adoptadog-logo.svg" alt="Adopt A Dog Logo - Seattle's premier dog adoption agency" width="320">
+          <h2 class="logo-text">Seattle's premiere dog adoption agency</h2>
+        </div>
+        <div class="col cart">
+          <p class="checkout"><img src="./images/cart-icon.svg" width="20" alt="shopping cart icon"><strong> $1234.56</strong></p>
+          <p class="checkout"><a href="./checkout.html" class="button green-button checkout-btn" >Checkout</a></p>
+        </div>
+      </div>
+      <div class="top-nav">
+        <nav class="top ">
+          <ul>
+            <li><a href="./index.html" class="active">Home</a></li>
+            <li><a href="./dogs.html">Dogs</a></li>
+            <li><a href="./blog.html">Blog</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section>
```

Comment:
I would suggest adding some padding or margin here to keep your content more to the center